user/customizing-the-build.md

```diff
@@ -268,6 +268,17 @@ git:
 ```
 {: data-file=".travis.yml"}
 
+## Git Clone Quiet
+
+Travis CI clones repositories without the quiet flag (`-q`) by default. Enabling the quiet flag can be useful if you're trying to avoid log file size limits or even if you just don't need to include it.
+
+You can enable the [quiet flag](https://git-scm.com/docs/git-clone#git-clone---quiet) in `.travis.yml`:
+
+```yaml
+git:
+  quiet: true
+```
+
```

Comment:
Not sure what it means but I think you need to had the following after your yaml snippet:

`{: data-file=".travis.yml"}`

See the examples above.

Reply:
okay, will add. didn't know what it did so didn't add it but will now.